suse_openstack_cloud

```diff
@@ -314,8 +314,10 @@ find_and_plog_files /var/log/ceph -type f
 #############################################################
 section_header "OpenStack config files"
 
-for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer manila barbican magnum trove designate sahara ironic; do
-    find_and_pconf_files /etc/$svc -type f -regex '.*\.\(co?nf\(ig\)?\|ini\|json\|filters\)'
+for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer manila barbican magnum \
+    monasca monasca-log-agent monasca-log-persister monasca-log-metrics monasca-log-transformer monasca-notification monasca-persister monasca-thresh \
```

Comment:
Thanks for breaking the lines, but they are still way too long, which somewhat defeats the point :-)  Could you keep them within 80 columns?